daemon/images/image_unix.go

```diff
@@ -1,15 +1,15 @@
 // +build linux freebsd
 
-package daemon // import "github.com/docker/docker/daemon"
+package images // import "github.com/docker/docker/daemon/images"
 
 import (
 	"runtime"
 
 	"github.com/sirupsen/logrus"
 )
 
-// getSize returns the real size & virtual size of the container.
-func (i *imageService) GetContainerLayerSize(containerID string) (int64, int64) {
+// GetContainerLayerSize returns the real size & virtual size of the container.
+func (i *ImageService) GetContainerLayerSize(containerID string) (int64, int64) {
```

Comment:
I understand why, but it feels odd that `GetContainerLayerSize()` is part of the image service